chocopud.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <style>
+        body
+        {
+            margin:5%;
+        }
+        p
+        {
+            margin-right: 30%;
+        }
+        img 
+        {
+            width: 200px;
+        }
+        ol
+        {
+            margin-right: 30%;
+        }
+    </style>
+</head>
+<body>
+    <a href="index.html">Home</a>
```

Comment:
As I mention below, style for functionality, distinguish :hover style.  Also, you must add more recipes, and it might be of use to allow the user to access the recipes from here as well.